article-label-to-content-tag/.gitignore

```diff
@@ -0,0 +1,8 @@
+.vscode/
+__pycache__/
+*.pyc
+build/
+dist/
+.eggs/
+*.egg-info/
+*.dist-info/
```

Comment:
I think we can drop this file.

Reply:
It's not important to me, I just like to have .gitignore files so I don't have to worry about adding IDE metadata or build artifacts etc to the repo.

Reply:
Sound good. Let's leave it in.